docs/extensions/tutorials/_posts/1970-01-14-SettingUpInstagram.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: doc
+permalink: /docs/extensions/tutorials/setting-up-instagram
+title: Setting up Instagram
+section: Tutorials
+---
+
+# Setting up Instagram
+
+In order to show an Instagram feed in a Shoutem app using the Photos RSS, you will have to set up a redirect URL. The first step is to go to the Instagram developer [site] and create a new client if you don't already have one. Take note of the Client ID and Client Secret, you will be needing these. In `Valid redirect URIs` insert `https://new.shoutem.com`.
```

Comment:
- Please explain that Instagram has closed off their API and now you can only access your own feed through API, so you'll only be able to add a feed which you own into your `Photos RSS` shortcut.
- `using the Photos RSS extension`

Reply:
Explain that if they don't have a developer client, they'll need to create one.

Instead of just telling them to go to `Valid redirect URIs`, explain that it's under the `Security` tab

Add some images to make the process clearer, like:
https://imgur.com/a/3M9ck
https://imgur.com/a/CjRGP